### PR TITLE
(maint) Message and string fixes

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

File: locales/pdk.pot [hunk 1]
```diff
@@ -227,26 +229,46 @@ msgstr ""
 msgid "Expected execution context to be :system or :module but got '%{context}'."
 msgstr ""
 
-#: ../lib/pdk/cli/exec.rb:155
+#: ../lib/pdk/cli/exec.rb:126
+msgid "PUPPET_GEM_VERSION is not supported by PDK. Please use the --puppet-version option on your PDK command or set the PDK_PUPPET_VERSION environment variable instead"
```
Comment (jbondpdx) [inline locales/pdk.pot]: Drop the "Please", please. :) And end the sentence with a period.

File: locales/pdk.pot [hunk 2]
```diff
@@ -814,10 +899,22 @@ msgstr ""
 msgid "A task named '%{name}' already exists in this module; defined in %{file}"
 msgstr ""
 
-#: ../lib/pdk/module/build.rb:149
+#: ../lib/pdk/module/build.rb:127
+msgid "%{message} Please rename the file or exclude it from the package by adding it to the .pdkignore file in your module."
```
Comment (jbondpdx) [inline locales/pdk.pot]: Remove the "Please".

**Comment by DavidS** [comment]
@jbondpdx no worries, I've got so many plates spinning, one more or less doesn't make a dent.

Updated the messages and rebased the PR to current master.

**Comment by glennsarti** [comment]
@DavidS Looks like this needs a rebase on master :-(

**Comment by DavidS** [comment]
Done.

**Pull request merged**